exchanges/gateio/gateio_test.go

```diff
@@ -3538,3 +3541,23 @@ func TestGetHistoricalFundingRates(t *testing.T) {
 
 	assert.NotEmpty(t, history, "should return values")
 }
+
+func TestGetOpenInterest(t *testing.T) {
+	t.Parallel()
+	_, err := g.GetOpenInterest(context.Background(), key.PairAsset{
+		Base:  currency.ETH.Item,
+		Quote: currency.USDT.Item,
+		Asset: asset.USDTMarginedFutures,
+	})
+	assert.ErrorIs(t, err, asset.ErrNotSupported)
+
+	_, err = g.GetOpenInterest(context.Background(), key.PairAsset{
```

Comment:
I will stop mentioning but can we assert that data is returned and that it is not empty.

Reply:
I will do this, but now I will ask you of this for all your tests 🌞

Reply:
I have caused pain for both of us. 😭